src/Controller/Component/AuthorizationComponent.php

```diff
@@ -31,16 +34,26 @@
 class AuthorizationComponent extends Component
 {
 
+    /**
+     * Constant for all actions config key.
+     */
+    const ALL = '*';
```

Comment:
I worry that this may lead to people accidentally opening up their actions by accident. Forcing the use of a whitelist forces people to be specific on which actions do not have authorization checks applied.

Reply:
It is a fair point. I'm wondering if we should do the same for model authorization. I think that model authorization should be done occasionally anyway (for methods that does not change state of existing entities or that query the db) so it would be easier to list only those actions and leave the rest for manual authorization.

Reply:
I am :+1: for opt-in features for authorization. Keeping the potential surprise count low is important to me.